taskcluster/kinds/all-pr-pipeline/kind.yml

```diff
@@ -23,13 +23,13 @@ kind-dependencies:
     - analyze-mono
 
 tasks:
-    all-pr:
+    all-pr-pipeline:
```

Comment:
I think it's fine to keep this change either way, seeing as it does not in fact cause _all_ tasks to run even now (just pipeline ones, as this is making clear!). It's probably worth amending the `all` kind with the same suffix if you keep this though.

Reply:
I renamed the `all*` task names to `all*-pipeline`.